analytics/build.gradle

```diff
@@ -30,7 +31,12 @@ dependencies {
   // with Android.
   //noinspection GradleDependency
   testImplementation 'org.assertj:assertj-core:1.7.1'
+  implementation "androidx.core:core-ktx:+"
```

Comment:
Should the dependencies be pinned to a specific version instead? This way there are no surprises when the API changes in the new updates.

Also, it seems like this dependency isn't actually required right now.